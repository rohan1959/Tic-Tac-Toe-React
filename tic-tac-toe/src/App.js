import './App.css';
import Board from './components/board/board';
function App() {
  return (<div>
    <h1> Tic Tac Toe</h1>
    <div className='box'>
      <Board></Board>
    </div>
  </div>)
}

export default App;
