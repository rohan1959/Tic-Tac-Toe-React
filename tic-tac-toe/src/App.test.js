import { render, screen } from '@testing-library/react';
import App from './App';
import { shallow } from 'enzyme';

test('renders header with Lets Play Tic Tac Toe', () => {
  render(<App />);
  const linkElement = screen.getByText(/Tic Tac Toe/i);
  expect(linkElement).toBeInTheDocument();
});
test('render app with tic-tac-toe layout', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
