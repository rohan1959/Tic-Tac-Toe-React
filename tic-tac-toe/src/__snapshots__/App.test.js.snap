// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app with tic-tac-toe layout 1`] = `
<div>
  <h2
    className="header"
  >
    Lets Play Tic Tac Toe !
  </h2>
  <div
    className="container"
  >
    <div
      className="box"
    >
      <div
        className="container-vertical"
      >
        <div
          className="status"
          id="status"
        >
          Player to make move is X
           
        </div>
        <div
          className="center"
        >
          <div
            className="board"
          >
            <div
              className="board-row"
            >
              <button
                className="square"
                onClick={[Function]}
              />
              <button
                className="square"
                onClick={[Function]}
              />
              <button
                className="square"
                onClick={[Function]}
              />
            </div>
            <div
              className="board-row"
            >
              <button
                className="square"
                onClick={[Function]}
              />
              <button
                className="square"
                onClick={[Function]}
              />
              <button
                className="square"
                onClick={[Function]}
              />
            </div>
            <div
              className="board-row"
            >
              <button
                className="square"
                onClick={[Function]}
              />
              <button
                className="square"
                onClick={[Function]}
              />
              <button
                className="square"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
        <div
          className="status center"
          id="summary"
        >
           
           
        </div>
        <div
          className="game-options"
        >
          <button
            className="btn btn-reset"
            id="reset"
            onClick={[Function]}
          >
            Reset
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
